### PR TITLE
Di services refactoring part 2

### PR DESCRIPTION
**Pull request by hendrikebbers**
all missing services refactored

### CODE REVIEW

**Review by qnswirlds — DISMISSED** [review]
LGTM!

**Review by Nana-EC — DISMISSED** [review]

File: hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/prehandle/DummyPreHandleDispatcher.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import edu.umd.cs.findbugs.annotations.NonNull;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+/**
+ * @deprecated This class is only needed to have a PreHandleDispatcher implementation that can be provided by dagger.
+ */
+@Deprecated
+@Singleton
+public class DummyPreHandleDispatcher implements PreHandleDispatcher {
```
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/prehandle/DummyPreHandleDispatcher.java:31]: Is this different from adding @Provides method ?

Comment (Neeharika-Sompalli) [inline hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/prehandle/DummyPreHandleDispatcher.java:31]: Why not use `PreHandleDispatcherImpl` ?

Comment (Neeharika-Sompalli) [inline hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/prehandle/DummyPreHandleDispatcher.java:31]: Will this be deleted once we have `PrehandleDispatcherImpl` ?

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileHandlers.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+    private final FileGetContentsHandler fileGetContentsHandler;
+
+    private final FileGetInfoHandler fileGetInfoHandler;
+
+    private final FileSystemDeleteHandler fileSystemDeleteHandler;
+
+    private final FileSystemUndeleteHandler fileSystemUndeleteHandler;
+
+    private final FileUpdateHandler fileUpdateHandler;
```
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileHandlers.java:42]: Somehow codecov complains about less coverage on this file

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM ! other than few questions and code coverage warnings

File: hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/prehandle/DummyPreHandleDispatcher.java (hunk 1 above)
Comment (hendrikebbers) [reply hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/prehandle/DummyPreHandleDispatcher.java:31]: sure

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileHandlers.java (hunk 2 above)
Comment (hendrikebbers) [reply hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileHandlers.java:42]: mmh, no idea.

Comment (Neeharika-Sompalli) [reply hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileHandlers.java:42]: Probably needs more tests for each class with its `ClassNameTest`

**Review by Nana-EC — APPROVED** [review]

**Review by mhess-swl — APPROVED** [review]

**Pull request merged**